Source: Johannes Nixdorf <mixi@exherbo.org>
Upstream: pending, http://bugzilla.adiscon.com/show_bug.cgi?id=309
Reason: /usr/sbin/convert masks ImageMagick's /usr/bin/convert

--- libee-0.3.2/src/Makefile.am.old	2012-02-11 10:30:42.679874249 +0100
+++ libee-0.3.2/src/Makefile.am	2012-02-11 10:31:07.029745759 +0100
@@ -1,4 +1,4 @@
-sbin_PROGRAMS = convert
+sbin_PROGRAMS = libee-convert
 
 # Uncomment for debugging
 DEBUG = -g
@@ -37,8 +37,8 @@
 libee_la_LIBADD = $(LIBXML2_LIBS) $(LIBESTR_LIBS)
 libee_la_LDFLAGS = -version-info 0:0:0
 
-convert_SOURCES = convert.c
-convert_CPPFLAGS =  -I$(top_srcdir) $(LIBEE_CFLAGS) $(LIBESTR_CFLAGS) $(LIBXML2_CFLAGS)
-convert_LDADD = $(LIBEE_LIBS) $(LIBXML2_LIBS) $(LIBESTR_LIBS)
+libee_convert_SOURCES = convert.c
+libee_convert_CPPFLAGS =  -I$(top_srcdir) $(LIBEE_CFLAGS) $(LIBESTR_CFLAGS) $(LIBXML2_CFLAGS)
+libee_convert_LDADD = $(LIBEE_LIBS) $(LIBXML2_LIBS) $(LIBESTR_LIBS)
 
 include_HEADERS = 
